resourcename/hasparent_test.go

```diff
@@ -0,0 +1,72 @@
+package resourcename
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHasParent(t *testing.T) {
+	t.Parallel()
+	for _, tt := range []struct {
+		test     string
+		name     string
+		parent   string
+		expected bool
+	}{
+		{
+			test:     "valid parent and child",
+			name:     "shippers/1/sites/1",
+			parent:   "shippers/1",
+			expected: true,
+		},
+
+		{
+			test:     "empty parent",
+			name:     "shippers/1/sites/1",
+			parent:   "",
+			expected: true,
+		},
```

Comment:
Should this be `true`? Not sure what the consequences would be, and would probably be caught by other validation in each service, but it feel like this should be `false` at first glance.

Reply:
Ah, yes let's change it to false

Reply:
Fixed!